src/Shared/PackageManagers/NPMProjectManager.cs

```diff
@@ -280,7 +287,7 @@ public override async Task<PackageMetadata> GetPackageMetadata(PackageURL purl)
                     }
 
                     // author(s)
-                    JsonElement? authorElement = OssUtilities.GetJSONPropertyIfExists(versionElement, "author");
+                    JsonElement? authorElement = OssUtilities.GetJSONPropertyIfExists(versionElement, "_npmUser");
```

Comment:
I see that both `author` (at the top level for a package) and `_npmUser` (on each version) exist in the package metadata: https://github.com/npm/registry/blob/master/docs/responses/package-metadata.md

Could you elaborate a little on why "_npmUser" on the latest version is the preferred value to use? This is definitely better than the old code (the spec doesn't show an author element as existing on version metadata entries), but I'm curious about the semantics of the author for the package vs. the author for the version, if we should be recording both vs. just the author of the latest version, etc. etc.

Reply:
`_npmUser` is populated by the npm cli, it is the user that actually pushed the given version to the npm registry. `author` is populated in the `package.json` and could just be left empty. We could have both if we want to, and `_npmUser` is stored as a new `metadata.Publisher` value.